store.go

```diff
@@ -105,7 +106,7 @@ func (s *Store) getOrCreateTypedKey(k Key, v Value, kt KeyType) *BRecord {
 					}
 				}
 			}
-			br = thing.(*BRecord)
+			br = (*BRecord)(thing)
```

Comment:
Sorry, didn't mean to seem rude. I was busy with business fluff this week so didn't see your comments until you'd already solved the problem.